app/about/page.tsx

```diff
@@ -1,7 +1,11 @@
+import React from "react";
+import SocialMediaBanner from "../components/SocialMediaBanner";
```

Comment:
You should be able to use an alias path @/app/components/ instead of the .. here.

Reply:
[
![Screenshot 2023-11-18 at 6 03 34 PM](https://github.com/LuskinOIC/website/assets/43101229/75a4afbc-2f0f-4be7-9390-a654d0df8565)
](url)
It's giving me an error when I try to add the @ path.

Reply:
Never mind, found the problem.